src/effects/bounce.ts

```diff
@@ -9,22 +9,36 @@ export const bounce = buildEffect({
   requiresAnimation: true,
   params: [
     intParam({
-      name: 'Bounce Height',
-      description: 'Positive number',
-      defaultValue: (image) =>
-        image ? Math.floor(image.dimensions[1] / 10) : 10,
+      name: 'Bounce Height %',
```

Comment:
Probably worth changing this to be a sliderParam now that it's got defined bounds.